manifests/platform.yaml

```diff
@@ -77,7 +77,7 @@ spec:
       - artifactory.algol60.net/csm-docker/stable/k8s.gcr.io/sig-storage/csi-resizer:v1.3.0
       - artifactory.algol60.net/csm-docker/stable/quay.io/cephcsi/cephcsi:v3.5.1
       # cray-nexus
-      - artifactory.algol60.net/csm-docker/stable/nexus3:3.25.0-2
+      - artifactory.algol60.net/csm-docker/stable/docker.io/sonatype/nexus3:3.38.0
```

Comment:
This needs to remain at 3.25.0 for csm-1.2

Reply:
Fixed now thanks